### PR TITLE
Fix #54: Added Standalone CLI Tools for Kafka Backup

### LINKED ISSUE #54: Standalone `kafka-backup` tool
It would be great to have a wrapper around `kafka-connect-standalone` to do backups and restores.
The tricky part here is to be able to stop the tool as soon as the restore is done (related: #46 )

### PR DESCRIPTION
**Pull request by itadventurer**
Next Step: Use them in tests

### CODE REVIEW

**Comment by itadventurer** [comment]
Fixes also #46

**Comment by jay7x** [comment]
LGTM! Though, remember about SSL ;)

File: bin/restore-standalone.sh [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+fi
+
+if [ -z "$TOPICS" ]; then
+  echo "--topics is missing"
+  echo "$HELP"
+  exit 1
+fi
+
+if [ -z "$OFFSET_FILE" ]; then
+  OFFSET_FILE="$SOURCE_DIR/restore.offsets"
```
Comment (jay7x) [inline bin/restore-standalone.sh:148]: I see no usage of this file later in configs.. Is it expected?

File: bin/restore-standalone.sh [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+
+fi
+
+##################################### Start Connect Standalone
+"$CONNECT_BIN" "$WORKER_CONFIG" "$CONNECTOR_CONFIG" >>"$WORKING_DIR/log" &
+PID=$!
+tail -F "$WORKING_DIR/log" &
+PID2=$!
+sleep 5
+grep -q "All records read. Restore was successful" <(tail -F "$WORKING_DIR/log" 2>/dev/null)
```
Comment (jay7x) [inline bin/restore-standalone.sh]: Not sure I got this part right.. I see you're trying to find the line in the log and terminate after. Cannot get why 2 `tail -F` on same log file though... 
I'd write it in this way:
```
tail -F "$WORKING_DIR/log" | while read -r line; do
  echo "$line" | grep -q "All records read. Restore was successful" && _term
done
```

Here `tail -F | while read` would block until right log message found.

File: bin/restore-standalone.sh (hunk 1 above)
Comment (itadventurer) [reply bin/restore-standalone.sh:148]: Good catch. Thank you!

File: bin/restore-standalone.sh (hunk 2 above)
Comment (itadventurer) [reply bin/restore-standalone.sh]: The tail in line 230 is just printing the logs out to `stdout` such that the user can see it. The second tail is piping it to `grep`. I have tried to use tee in line 229 but then the `kill $PID` would not kill Kafka connect.

In general I do not like to use `while read` in bash. It feels awkward to me.

I have added a bunch more comments to clarify what happens. Maybe you have a better idea how to do so ;)

**Comment by itadventurer** [comment]
SSL should work out of the box by providing `--command-config` with a properties file. @jay7x maybe you could try it out? I do not have a cluster with SSL enabled here (too lazy to set it up :stuck_out_tongue: )

**Pull request merged**